src/components/TokenSelector.tsx

```diff
@@ -76,9 +76,10 @@ interface Props {
   tokens: TokenDetails[]
   selected: TokenDetails
   onChange: (selected: TokenDetails) => void
+  tabIndex?: number
```

Comment:
if you're setting a default value in the comp then set this as required

Reply:
But then, would the strict mode allow to create a <TokenSelector> component without providing the `tabIndex`?

Just tried and it doesn't work:

```
ERROR in /home/alfeto/work/dex-react/src/components/TradeWidget/TokenRow.tsx(192,8):
TS2741: Property 'tabIndex' is missing in type '{ label: string; isDisabled: boolean; tokens: TokenDetails[]; selected: TokenDetails; onChange: (selected: TokenDetails) => void; }' but required in type 'Props'.
```

Reply:
I'll merge this for now. I can push this change in a new PR later.